WcaOnRails/app/views/search_results/index.html.erb

```diff
@@ -77,6 +77,21 @@
       <%= paginate @posts, param_name: :posts_page, params: { anchor: "posts" } %>
     <% end %>
 
+    <% if !@regulations.empty? %>
+      <h2><%= anchorable "Regulations and Guidelines" %></h2>
+      <% @regulations.each do |reg| %>
+        <div>
+          <%= link_to reg["url"] do %>
+            <h4><%= wca_highlight(reg["id"], query_parts) %></h4>
+          <% end %>
+          <blockquote>
+            <%= wca_highlight(reg["text"], query_parts) %>
```

Comment:
FYI the links are rendered in the omnisearch "search as you type" field!
I'm fine with not rendering them, but we should probably be consistent and remove them in the rendering [here](https://github.com/cubing/worldcubeassociation.org/pull/722/files#diff-899eee8f20888bf9629deb12fd5852e9R81) too.
What do you think?

Once this last detail is figured out I'll merge and deploy it :)